rom/dev/src/fuse.rs

```diff
@@ -0,0 +1,55 @@
+/*++
+Licensed under the Apache-2.0 license.
+
+File Name:
+
+    fuse.rs
+
+Abstract:
+
+    The file contains Fuse-related Implementations.
+
+--*/
+use caliptra_common::{FuseLogEntry, FuseLogEntryId};
+use caliptra_drivers::{CaliptraError, CaliptraResult};
+use core::mem::size_of;
+use zerocopy::AsBytes;
+
+const FUSE_LOG_SIZE: usize = 1024;
```

Comment:
Ideally we would this this value from the linker script. But I can't really figure out how to make that work. Will keep thinking about it, but doesn't need to block this PR.